views/js/controller/admin/index.js

```diff
@@ -253,40 +267,41 @@ define([
             _installListeners('#permissions-table-users');
             _installListeners('#permissions-table-roles');
 
-            $form.on('submit', function (e) {
-                e.preventDefault();
-                e.stopImmediatePropagation();
-            });
-            $submitter.on('click', function (e) {
-                e.preventDefault();
-
-                if ($submitter.hasClass('disabled')) {
-                    return;
-                }
-
-                if (!_checkManagers('form')) {
-                    feedback().error(errorMsgManagePermission);
-                    return;
+            //find the old submitter and replace it with the new component
+            var taskCreationButton = taskCreationButtonFactory({
+                type : 'info',
+                icon : 'save',
+                title : __('Save'),
+                label : __('Save'),
+                taskQueue : taskQueue,
+                taskCreationUrl : $form.attr('action'),
+                taskCreationData : function getTaskCreationData(){
```

Comment:
May as well be anonymous, no need to name it IMO.
```suggestion
                taskCreationData : function() {
```

Reply:
@wazelin We have in style guide rule for ES5 https://oat-sa.atlassian.net/wiki/spaces/DEV/pages/79037064/TAO+Frontend+Coding+guide
`- named function expressions for methods : `{ method : function method() }``

Reply:
Do we have `.eslint` configurations, aligned with those?

Reply:
Oh, right, I see now.
Thank you!

Reply:
No, .eslint is configured for new ES6

Reply:
According to that guide, `getTaskCreationData()` should be `taskCreationData()`, though.
Modified my suggestion.

Reply:
yes, you are right